### PR TITLE
Cigarette Fixes

### LINKED ISSUE #5593: "NO ICON" - Body temperature?
https://puu.sh/BZNK2/c7a43ab675.png

Hovering over it tells me its body temperature. It appeared when no particular thing was happening. Room temperature was still a happy 20C.

### LINKED ISSUE #5594: Temperature breaking bodyscanners
Body scanners show up blank, when scanned with a handheld scanner, this shows up on temperature, seen it a few times now.
capture

### PR DESCRIPTION
**Pull request by BurgerLUA**
There was an issue with one of the travis checks that caused reagents with incorrect specific heat values to sneak into the game and cause havok. This fixes that.

Fixes #5594
Fixes #5593

### CODE REVIEW

**Comment by BurgerLUA** [comment]
!wip

**Comment by BurgerLUA** [comment]
!review

File: code/controllers/subsystems/chemistry.dm [hunk 1]
```diff
@@ -52,7 +52,7 @@ var/datum/controller/subsystem/chemistry/SSchemistry
 			chemical_reagents[R.id].specific_heat = final_heat
 			return final_heat
 
-	if(R.fallback_specific_heat)
+	if(R.fallback_specific_heat > 0)
```
Comment (Sindorman) [inline code/controllers/subsystems/chemistry.dm]: I don't think it is necessary as any integeter of 1 and above will return true

Comment (BurgerLUA) [reply code/controllers/subsystems/chemistry.dm]: This is actually false and the cause of the issue.

if(-1) returns true. Check discord for proof.

**Comment by BurgerLUA** [comment]
Btw the travis issue wasn't because I was a bad coder but because I had bad eyesight. There was a mislabeled reagent from all the way back in 2015 that caused issues.

**Pull request merged**